src/mmw/js/src/core/views.js

```diff
@@ -83,6 +84,33 @@ var HeaderView = Marionette.ItemView.extend({
 
 });
 
+// Init the locate plugin button and add it to the map.
+function addLocateMeButton(map) {
+    var locateOptions = {
+        position: 'topright',
+        metric: false,
+        drawCircle: false,
+        showPopup: false,
+        follow: true,
+        markerClass: L.marker,
+        markerStyle: {
+            opacity: 0.0,
+            clickable: false,
+            keyboard: false
+        },
+        locateOptions: {
+            maxZoom: 9,
+            // Cache location response, in ms
+            maximumAge: 60000
```

Comment:
If we're using this value twice, it should probably end up in a single var.  Probably the same with `maxZoom` above, but I'm less clear as to what it does and we're using zoom 10 below.